conda_rpms/install.py

```diff
@@ -399,6 +401,117 @@ def symlink_conda(prefix, root_dir):
     if not os.path.lexists(prefix_deactivate):
         os.symlink(root_deactivate, prefix_deactivate)
 
+
+# ========================== begin noarch functions =========================
+
+python_entry_point_template = dedent("""
+# -*- coding: utf-8 -*-
+import re
+import sys
+from %(module)s import %(import_name)s
+if __name__ == '__main__':
+    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
+    sys.exit(%(func)s())
+""").lstrip()
+
+
+def get_python_noarch_target_path(source_short_path, target_site_packages_short_path):
+    """
+    Modified version of conda/common/path.py:get_python_noarch_target_path at
+    be8c08c083f4d5e05b06bd2689d2cd0d410c2ffe
+        * removed windows specific support
+
+    """
+    if source_short_path.startswith('site-packages/'):
+        sp_dir = target_site_packages_short_path
+        return source_short_path.replace('site-packages', sp_dir, 1)
+    elif source_short_path.startswith('python-scripts/'):
+        bin_dir = 'bin'
+        return source_short_path.replace('python-scripts', bin_dir, 1)
+    else:
+        return source_short_path
+
+
+def compile_pyc(python_exe_full_path, py_full_path):
+    """
+    Modified version of conda/gateways/disk/create.py/compile_pyc at
+    be8c08c083f4d5e05b06bd2689d2cd0d410c2ffe
+        * simplified something
+        * removed check that the pyc file already exists
+        * remove check that pyc file has been created after the compilation
+
+    """
+    command = [python_exe_full_path, '-Wi', '-m', 'py_compile', py_full_path]
+    subprocess.call(command)
+
+    # TODO: should we check the pyc file now exists. The location is different for
+    # py2/3???
```

Comment:
Check logging instead? Then again, how do we get access to the log output, and when would we check it ... dont' know.

Reply:
There was a `log.trace` that would put the output of the subprocess.call in the log. 
I had initially removed that but I will put it back in

Reply:
okay. log.trace not supported.
I've changed it to log.info